src/main/java/seedu/recipe/model/Model.java

```diff
@@ -184,6 +184,12 @@
     /** Returns an unmodifiable view of the filtered record list */
     ObservableList<Record> getFilteredRecordList();
 
+    /**
+     * updates goal tally every time a record is added
+     * @param record
```

Comment:
I think it might be better to standardize the JavaDoc convention and follow AB3's format, which refers to parameters as {@code record} instead of @param record